controller/api/v1alpha1/prautomation_types.go

```diff
@@ -110,6 +111,10 @@ func (in *PrAutomation) SetCondition(condition metav1.Condition) {
 
 // PrAutomationSpec ...
 type PrAutomationSpec struct {
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:validation:Enum=CLUSTER;SERVICE;PIPELINE;UPDATE;UPGRADE
+	Role *console.PrRole `json:"role,omitempty"`
```

Comment:
@michaeljguarino While working on it I have noticed that it was not exposed in the CRD. Should I keep the change?

Reply:
Yeah that could be nice.

Reply:
Ok, I'll keep it.